.readthedocs.yaml

```diff
@@ -12,9 +12,7 @@ build:
     python: "3.10"
   jobs:
     pre_build:
-      - cd docs
       - sh examples_to_rst.sh
```

Comment:
Would it not work if we place the `examples_to_rst.sh` into either `docs` or `examples` and call it `sh docs/examples_to_rst.sh` in the .readthedocs.yaml?

Reply:
Yes, it should work. Let me implement it